test/Analysis/test-allocation.mlir

```diff
@@ -179,28 +179,28 @@ tt.func @longlive(%A : !tt.ptr<f16>) {
 tt.func @multi_color(%A : !tt.ptr<f16>) {
   // CHECK: offset = 0, size = 64
   %cst = triton_gpu.local_alloc : () -> !tt.memdesc<4x8xf16, #A_SHARED, #triton_gpu.shared_memory>
-  // CHECK-NEXT: offset = 1536, size = 32
+  // CHECK-NEXT: offset = 256, size = 32
   %cst_0 = triton_gpu.local_alloc : () -> !tt.memdesc<4x4xf16, #A_SHARED, #triton_gpu.shared_memory>
   // CHECK-NEXT: offset = 1664, size = 128
   %cst_1 = triton_gpu.local_alloc : () -> !tt.memdesc<16x4xf16, #A_SHARED, #triton_gpu.shared_memory>
   %cst_2 = arith.constant dense<0.000000e+00> : tensor<16x32xf16, #AL>
-  // CHECK-NEXT: scratch offset = 128, size = 1152
+  // CHECK-NEXT: scratch offset = 128, size = 2
```

Comment:
Where do the 2 bytes come from?

Reply:
No idea, TBH.  You can see that I'm reporting 0 bytes of shmem to the allocator.

Reply:
Interesting. I'll take a look tomorrow

Reply:
Yes this is concerning. Also Since the codegen now detects that this is a noop convert_layout this kind of defeats the purpose of this test. I wonder if we should update the test to convert to a different layout to make sure the shared memory is accounted for.

Reply:
OK, I've modified the test to avoid the nop conversions.  Now the allocations are the same as before.